_data/projects/triplea.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: triplea
+desc: war game in java
+site: https://github.com/triplea-game/triplea
+tags:
+- collaboration
+- java
+- war
+- game
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/triplea-game/triplea/labels/up-for-grabs
```

Comment:
This link should point to a list of labels, but I'm not sure which one you are using in the issue tracker